src/app/(dashboard)/levels/[entry]/sortable-multipliers.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+
+import { RiArrowDownSLine, RiArrowUpSLine } from "@remixicon/react";
+import { useState } from "react";
+import { SidebarSection } from "@/components/leaderboard/SidebarSection.tsx";
+import { type Multiplier, MultiplierDisplay } from "./04-multiplier.tsx";
+
+interface SortableMultipliersProps {
+	multipliers: Multiplier[];
+}
+
+export function SortableMultipliers({ multipliers }: SortableMultipliersProps) {
+	const [sortAscending, setSortAscending] = useState(true);
+
+	if (multipliers.length === 0) {
+		return null;
+	}
+
+	const sortedMultipliers = [...multipliers].sort((a, b) => {
+		return sortAscending ? a.multiplier - b.multiplier : b.multiplier - a.multiplier;
+	});
+
+	const toggleSort = () => {
+		setSortAscending(!sortAscending);
+	};
```

Comment:
Use `function` syntax instead. For the state, pass a callback as argument instead and toggle the `previousValue` there